Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:24.04 AS uv_base
+FROM ubuntu:25.10 AS uv_base
```

Comment:
Is that really mandatory ?
Development and bug-fix efforts on non-LTS versions are known to be lower.